test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -109,34 +168,40 @@ def test_delete_default_sched(self):
         """
         Delete default scheduler.
         """
-        a = {}
         try:
-            rc = self.server.manager(MGR_CMD_DELETE, SCHED, a, id="main")
+            self.server.manager(MGR_CMD_DELETE,
+                                SCHED,
+                                id="main")
```

Comment:
You're hard coding the name of the default scheduler.  This is yet another reason you shouldn't allow it to be changed.  This test will fail if the default scheduler name has changed from the default.